client/landing/gutenboarding/components/domain-picker/list.tsx

```diff
@@ -37,16 +56,28 @@ const DomainPicker: FunctionComponent< Props > = ( {
 					/>
 				</PanelRow>
 
+				<HorizontalRule className="domain-picker__divider" />
+
 				{ suggestions?.length ? (
-					<PanelRow>
-						<h3 className="domain-picker__recommended-header">{ NO__( 'Recommended' ) }</h3>
-						<ul>
-							{ suggestions.map( ( { domain_name } ) => (
-								<li key={ domain_name }>{ domain_name }</li>
-							) ) }
-						</ul>
+					<PanelRow className="domain-picker__panel-row">
+						<div className="domain-picker__recommended-header">{ NO__( 'Recommended' ) }</div>
+						{ suggestions.map( suggestion => (
+							<Button
+								onClick={ handleDomainPick( suggestion ) }
+								className="domain-picker__suggestion-item"
+								key={ suggestion.domain_name }
+							>
+								<div className="domain-picker__suggestion-item-name">
+									{ suggestion.domain_name }
+								</div>
+								<div className="domain-picker__suggestion-action">{ NO__( 'Upgrade' ) }</div>
```

Comment:
`<div />`s within `<button />` :grimacing: 

> [`button`](https://html.spec.whatwg.org/dev/form-elements.html#the-button-element) should have non-interactive [phrasing content](https://html.spec.whatwg.org/dev/dom.html#phrasing-content). `div` children [...] violate this 😞 

(https://github.com/Automattic/wp-calypso/pull/37664#discussion_r347848176)

Reply:
Maybe we can turn them into spans? :thinking:

Reply:
Seems to work: 46ebf33